addons/delivery/wizard/choose_delivery_carrier_views.xml

```diff
@@ -38,7 +38,21 @@
                 </div>
                 <footer>
                     <button name="button_confirm" invisible="not context.get('carrier_recompute')" type="object" string="Update" class="btn-primary" data-hotkey="q"/>
-                    <button name="button_confirm" invisible="context.get('carrier_recompute')" type="object" string="Add" class="btn-primary" data-hotkey="q"/>
+                    <button
+                        name="button_confirm"
+                        type="object"
+                        invisible="context.get('carrier_recompute') or not delivery_type in ('fixed', 'base_on_rule')"
```

Comment:
```suggestion
                        invisible="context.get('carrier_recompute') or (delivery_type not in ('fixed', 'base_on_rule') and not display_price)"
```
After clicking on 'Get rates' with Fedex, I should be able to click on the button ...
![image](https://github.com/user-attachments/assets/a0391528-bd5d-468a-883c-f5b23a71d881)